build.gradle

```diff
@@ -17,7 +17,7 @@ repositories {
 }
 
 allprojects {
-    version = '2.6.0'
+    version = '2.7.0'
```

Comment:
We have to wait until 2.7.0 is released or consume 2.7.0-SNAPSHOT. 

I would recommend moving to SNAPSHOT version which syncs up with bleeding edge of OpenSearch.

Reply:
Updated.